performance_regression_test.py

```diff
@@ -699,6 +699,8 @@ def test_mixed_with_stairs(self):
         # allow to correctly save results for future compare
         self.create_test_stats(doc_id_with_timestamp=True)
         self.run_fstrim_on_all_db_nodes()
+        for node in self.db_cluster.nodes:
+            node.run_nodetool("disableautocompaction keyspace1 standard1", ignore_status=True)
```

Comment:
I don't think it's something we want to do.
Is it something Peter or others did in their tests??